ui/pages/swaps/searchable-item-list/index.scss

```
@@ -54,7 +54,7 @@
     padding: 8px 12px;
     box-sizing: border-box;
     cursor: pointer;
-    border-top: 1px solid var(--Grey-100);
+    border-top: 1px solid var(--color-border-default);
```

border-muted should be used for any Grey100 that is a border
```suggestion
    border-top: 1px solid var(--color-border-muted);
```